### DISCUSSION TITLE
Question about `actions.change_dir.global`

### DISCUSSION THREAD

**Discussion by ferdinandrau** [nvim-tree/nvim-tree.lua#2971] [Q&A] [answered]
In the docs, it says:

```vimdoc
*nvim-tree.actions.change_dir.global*
Use `:cd` instead of `:lcd` when changing directories.
Consider that this might cause issues with the
|nvim-tree.sync_root_with_cwd| option.
  Type: `boolean`, Default: `false`
```

Im wondering what **issues** this is talking about? The behavior i want is this:
- Changing directory in the tree with Ctrl + ] updates nvim's cwd globally (i.e. does `:cd`)
- doing `:cd` manually updates the tree view to the new cwd

And I can achive this exact behavior by setting `actions.change_dir.global = true` and `sync_root_with_cwd = true` at the same time. There does not seem to be any issue. Why does it say the two options are incompatible?

**Comment by alex-courtis** [comment, marked as answer]
Good find! I also use those two options and don't encounter any issues.

That note was added ~3 years ago when the option was added and has not changed since then.

Removing.
